### ISSUE TITLE
Option for Read-Only access

### ISSUE THREAD

**Issue by funkybob** [integrations/slack#1049]
From reading older issues I can see previously this integration only required read permissions, but due to added features that was expanded to also needing write.

I don't want to permit this service to "Act on mybehalf", but am given no option.

Is it feasible for a read-only integration, or should I look for a different product?

**Comment by mike-marcacci** [comment]
I agree that this is an important feature.  While this currently only supports opening and closing issues and PRs (and is thus reasonably innocuous) it provides a real way to bypass access controls set up by the organization by GitHub. Any user in a Slack channel with the GitHub integration – including out-of-organization guests – can take these actions through Slack regardless of their permissions within GitHub. Having the ability to limit this functionality is very important, especially if the features of this integration continue to grow.

**Comment by ataylorme** [comment]
+1. The application requires `read` and `write` access for Actions to use the Actions notifications feature. We would like to use this feature but are not willing to grant `write` access.
